source/MaterialXRuntime/RtStrings.cpp

```diff
@@ -0,0 +1,79 @@
+//
+// TM & (c) 2020 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#include <MaterialXRuntime/RtStrings.h>
+
+namespace MaterialX
+{
+
+namespace RtStrings
+{
+    const RtString DOC("doc");
```

Comment:
That's a good point. The main reason why I used a namespace for this in the beginning is that with a namespace we can spread this into multiple files. So in theory string constants could be declared more locally to where they are used but still within a common namespace. However, we are not using this yet and I doubt this will ever be used, so I like your suggestion of keeping it all inside the RtString class instead. I will fix this.